balloc/balloc.c

```diff
@@ -403,6 +405,10 @@ static int balloc_group_info_init(struct m0_balloc_group_info *gi,
 static void balloc_group_info_fini(struct m0_balloc_group_info *gi)
 {
 	m0_mutex_fini(bgi_mutex(gi));
+	m0_mutex_fini(&gi->bgi_bac_mutex);
+	m0_list_fini(&gi->bgi_bac_head);
+	m0_chan_fini_lock(&gi->bgi_chan);
+	m0_mutex_fini(&gi->bgi_chan_mutex);
 	m0_list_fini(&gi->bgi_normal.bzp_extents);
```

Comment:
These lists should be replaced by m0_tlist.

This can be done in another ticket.